en/docs/develop/product-apis/getting-started/guide-gateway-v1.md

```diff
@@ -0,0 +1,63 @@
+# Getting Started
+
+WSO2 API Manager is a complete API Management solution that addresses all aspects of API management including API development, productization, consumption, security, rate limitation, monetization and analytics.
+
+
+The REST API is implemented based on the REST specification and best practices as a CXF REST web application running on WSO2 API Manager. A contract-first approach could be utilized for API development starting with a Open API specification. 
+See the Gateway API's [Swagger definition](https://raw.githubusercontent.com/wso2/carbon-apimgt/v6.6.163/components/apimgt/org.wso2.carbon.apimgt.rest.api.gateway.v1/src/main/resources/gateway-api.yaml) , which is written using Open API 3.0 .
+Alternatively, this API definition can be accessed using the product URL as well `https://<host-name[:port]>/api/am/gateway/v1/swagger.json`.
+
+
+The API comes with a pluggable security mechanism. Since API security is implemented as a CXF handler, if you need to plug a custom security mechanism, you can write your own handler and add it to the web service.
+
```

Comment:
Add this section "In the default flow, when an API is invoked with an access token, first the CXF handler gets engaged and calls an introspection API to validate the token"

Reply:
https://github.com/wso2/docs-apim/pull/1527/commits/dfc6cce75030fb2c093887ef3c5342fc7a9bce21